net-firewall/iptables/iptables-1.8.7.ebuild

```diff
@@ -41,10 +42,7 @@ BDEPEND="${BUILD_DEPEND}
 	)
 "
 RDEPEND="${COMMON_DEPEND}
-	${BUILD_DEPEND}
```

Comment:
Why dropping it? I'd try to explain it either here or in a readme file in this directory.

Reply:
There is a comment about it in the fixup commit. I haven't noticed that when I reviewed this commit.